resources/css/_code.css

```
@@ -15,6 +15,10 @@ pre code.torchlight .summary-caret {
     @apply mr-1;
 }
 
+code {
```

I think this would make things a bit more explicit (untested)

```suggestion
p code {
```